src/Provider/ThemeProvider.jsx

```diff
@@ -1,9 +1,28 @@
+import { blueGrey, grey } from '@mui/material/colors';
 import { createTheme } from '@mui/material/styles';
 
-export const theme = createTheme({
+export const lightTheme = createTheme({
   palette: {
     secondary: {
       main: '#854d27',
     },
   },
 });
+
+export const darkTheme = createTheme({
+  palette: {
+    mode: 'dark',
+    secondary: {
+      main: grey[300] 
+    },
+    text: {
+      primary: blueGrey[900],
+      secondary: grey[50]
+    },
+    background: {
+      default: grey[900],
+      paper: grey[200],
+    },
```

Comment:
For all MUI control we can add CSS for different Modes(Dark/light) here as well to make it generic